src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -564,6 +568,11 @@ def launch_fluent(
             return new_session.create_from_server_info_file(
                 server_info_filepath, cleanup_on_exit, start_transcript
             )
+        except Exception as ex:
+            identifier = "fluent.exe"
+            details = "\n" + "Fluent Launch Path: " + launch_string.split(identifier)[0] + identifier
```

Comment:
Looks like launch_string is the only argument we need to pass to the LaunchFluentError constructor. Then we can put this code there and have just one simple line inside the except block.

Reply:
Done, Thank you for the suggestion. Looks much better now.